.golangci.yml

```diff
@@ -11,9 +11,14 @@ linters:
     - sql
     - module
     - unused
+  enable:
+    - predeclared
```

Comment:
This also disallows redefining predeclared variables like `true` etc. This is a valid Go code:

```go
true = false
```

Yeah, nasty.